tests/reg_tests/test_amg.py

```diff
@@ -82,6 +82,27 @@
             "innerPreconIts": 3,
         },
     },
+    {
```

Comment:
Seems like we are only checking convergence for these tests. These values here may not reflect best practice, but do we not want to track some reference values, either convergence rate (or some functional output) as well (or both)? If not now, we should add such tests in the future (create issue with improvements).

Reply:
I'm not sure if it's worth checking functional outputs because that doesn't really test the solver. For convergence rate, we could track the number of iterations or time, but both of those will likely depend on the image the test is running on.

Reply:
Right, I understand, I will approve this for now. However, I do think we need to improve the tests sooner than later (especially we have an open [issue](https://github.com/mdolab/adflow/issues/324) on the AMG). 
@sseraj can you maybe create an issue, suggesting updates that we can make to improve these solver tests?

Reply:
I opened an issue here: #361